numpy/core/setup.py

```diff
@@ -605,6 +606,12 @@ def generate_api(ext, build_dir):
     config.add_include_dirs(join('src', 'umath'))
     config.add_include_dirs(join('src', 'npysort'))
 
+    # Need to add location of npy_math_internal.h, generated from src/npymath/npy_math_internal.h.src
+    npymath_build_dir = join('build', 'src.%s-%s.%s' % (sysconfig.get_platform(), sys.version_info[0], sys.version_info[1]),
```

Comment:
ah this is npymath which might not have the correct setup to do so. Copying the method from multiarray and umath module may be nicer.